mega-linter-runner/lib/runner.js

```diff
@@ -124,8 +124,8 @@ ERROR: Docker engine has not been found on your system.
     if (options["remove-container"]) {
```

Comment:
This doesn't really work. The `optionater` converts hyphen-separated options to _camelCase_ property instead of _kebab-case_. I know I was the one to introduce this bug, but since you already have the PR, I kindly ask you to apply these changes:
```suggestion
    if (options.removeContainer) {
```

Reply:
Thank you for the catch. This must be the cause of #1917. I've opened #1928 to address this.